zkevm-circuits/src/state_circuit/lexicographic_ordering.rs

```diff
@@ -323,19 +318,19 @@ impl<F: Field> Queries<F> {
     }
 }
 
-fn rw_to_be_limbs(row: &Rw) -> Vec<u16> {
-    let mut id = row.id().unwrap_or_default() as u32;
+fn rw_to_be_limbs<F: Field>(row: &RwRow<F>) -> Vec<u16> {
```

Comment:
Could you add a TODO here stating that the order from the spec is: `tag, id, address, field_tag, storage_key, rw_counter`?
https://github.com/privacy-scaling-explorations/zkevm-specs/blob/cba1e81ce323a6be8fb0ad512269c5e54549547d/src/zkevm_specs/state.py#L487